index.js

```diff
@@ -1,18 +1,133 @@
-const readline = require('readline');
-const rl = readline.createInterface(process.stdin, process.stdout);
+//share files within project
+const lib = require('./ask');
```

Comment:
Just a quick note here, you could pull just the ask function in so you wouldn't have to call lib.ask by using the following:

`const { ask } = require('./ask')`

Nothing major, but saves a bit of code in the project, and helps with readability.